plugins/http.js

```diff
@@ -1,6 +1,9 @@
 export default function ({ $http }) {
     const token = process.env.AUTH_TOKEN || ''
     $http.setToken(token, 'Token')
+    if (process.env.DEPLOY_ENV) {
```

Comment:
How does this resolve 404 issue? What's the root cause and why does backend need it?

Reply:
Currently, BE specify [ALLOW_HOST](https://github.com/pycontw/pycon.tw/blob/03d3e8af584d4b3d3bcbdd23fcf05d6da70f93de/src/pycontw2016/settings/production/base.py#L19) here.
That means that we can only fetch the api with specific host header, or we will encounter 400 issue.

Reply:
then why the value of the header is `process.env.DEPLOY_ENV` rather than the host name?

Reply:
How should DEPLOY_ENV be provided? It's not a nuxt/vue built-in env var, right?

Reply:
The naming might confuse you.
The better naming env might be `process.env.HOST_NAME`.

Reply:
> How should DEPLOY_ENV be provided? It's not a nuxt/vue built-in env var, right?

Yes, we should add this into the machine .env, just like we did in `AUTH_TOKEN`

Reply:
ok, maybe rename and add it to `.env.sample` (and maybe add code comment or explain it somewhere in case we forget why we need this)

Reply:
fixed.